src/rules/isort/rules/organize_imports.rs

```diff
@@ -112,8 +124,8 @@ pub fn organize_imports(
         Location::new(range.location.row(), 0),
         Location::new(range.end_location.row() + 1 + num_trailing_lines, 0),
     );
-    let actual = dedent(locator.slice_source_code_range(&range));
-    if actual == dedent(&expected) {
```

Comment:
(IIRC, we used to compare `indent(&expected, indentation)` to `actual`, but `indent` and `dedent` don't respect `stylist.line_ending`, and so files with CLRF newlines would always fail the comparison.)